docs/index.yaml

```diff
@@ -14,28 +14,13 @@ entries:
     - https://nominatim.org/
     - https://github.com/peter-evans/nominatim-k8s
     urls:
-    - https://jobvite.github.io/nominatim-k8s/nominatim-3.5.1.tgz
-    version: 3.5.1
+    - https://jobvite.github.io/nominatim-k8s/docs/nominatim-3.5.2.tgz
```

Comment:
shouldn't include docs...in the repo doc's is a basepath for the jobvite.github.io its not a subdir so need to run index without the added docs info